ui/src/main/webapp/src/app/executions/executions-list.component.html

```diff
@@ -27,7 +27,10 @@ <h2 i18n="heading|executions list">Executions</h2>
         <td *ngIf="!execution.timeCompleted"></td>
         <td>
             <a class="link cancel" *ngIf="canCancel(execution)" (click)="cancelExecution(execution)" i18n="button">Cancel</a>
-            <a class="link" *ngIf="execution.state == 'COMPLETED' && execution?.analysisContext?.generateStaticReports" target="_blank" href="{{formatStaticReportUrl(execution)}}"> View Static Report </a>
+            <a
+                    class="link"
+                    *ngIf="execution.state == 'COMPLETED'"
+                    [routerLink]="['/projects', execution.projectId, 'reports', execution.id]">View Reports</a>
```

Comment:
Wait, I commented on the wrong line... this is intentional in this case. The previous link went to the static reports, the new link goes to the dynamic reports.

The dynamic reports shouldn't be restricted based on that flag.